autogen/README.md

```diff
@@ -137,11 +137,8 @@ The [project factory](https://github.com/terraform-google-modules/terraform-goog
 - [kubectl](https://github.com/kubernetes/kubernetes/releases) 1.9.x
 #### Terraform and Plugins
 - [Terraform](https://www.terraform.io/downloads.html) 0.11.x
-{% if private_cluster %}
 - [terraform-provider-google-beta](https://github.com/terraform-providers/terraform-provider-google-beta) v2.3, v2.6, v2.7
-{% else %}
 - [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) v2.3, v2.6, v2.7
```

Comment:
Will removing this cause a breaking change to existing cluster deployments using the non-beta provider?

Reply:
will run more tests.

Reply:
This will not cause breaking changes. 3422f69 fixes setting default values that will not cause recreation of the cluster.